runtime/src/iree/hal/drivers/local_task/task_queue.h

```diff
@@ -22,27 +22,60 @@
 extern "C" {
 #endif  // __cplusplus
 
+// A single batch of command buffers submitted to a device queue.
+// All of the wait semaphores must reach or exceed the given payload value prior
+// to the batch beginning execution. Each command buffer begins execution in the
+// order it is present in the list, though note that the command buffers
```

Comment:
same nit about plural "values"
```suggestion
// All of the wait semaphores must reach or exceed the given payload values
// prior to the batch beginning execution. Each command buffer begins execution
// in the order it is present in the list, though note that the command buffers
```